server/channels/api4/plugin_test.go

```diff
@@ -121,6 +103,7 @@ func TestPlugin(t *testing.T) {
 		// Successful upload
 		manifest, _, err = client.UploadPlugin(bytes.NewReader(tarData))
 		require.NoError(t, err)
+		assert.Equal(t, "testplugin", manifest.Id)
```

Comment:
With `TestInstallMarketplacePluginPrepackagedDisabled` factored out, I needed to "use" this instance of manifest.